README.md

```diff
@@ -46,7 +46,7 @@ source /usr/local/bin/virtualenvwrapper.sh
 
 Run <code>source ~/.bashrc</code>
 
-Run <code>mkvirtualenv hotosm</code> to create the hotosm virtual environment.
+Run <code>mkvirtualenv --system-site-packages hotosm</code> to create the hotosm virtual environment.
```

Comment:
For access to `python-gdal` installed via `apt` (see :83). `GDAL` wouldn't install from source, even with the previous instructions in here (latest version may depend on 2.x now?).